### PR TITLE
build(relup): add new script for download relup base packages

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

**Review by terry-xiaoyu — APPROVED** [review]

File: .github/workflows/build_packages.yaml [hunk 1]
```diff
@@ -269,32 +265,6 @@ jobs:
         path: .
     - name: unzip source code
       run: unzip -q source.zip
-    - name: downloads old emqx zip packages
-      env:
-        PROFILE: ${{ matrix.profile }}
-        ARCH: ${{ matrix.arch }}
-        SYSTEM: ${{ matrix.os }}
-        OLD_VSNS: ${{ needs.prepare.outputs.old_vsns }}
```
Comment (zmstone) [inline .github/workflows/build_packages.yaml:277]: old versions are used as a matrix dimension, doing relup tests in a loop would mean longer time?

File: scripts/relup-base-packages.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+set -euo pipefail
+
+PROFILE="${1}"
+if [ "$PROFILE" = "" ]; then
+    $PROFILE="emqx"
+fi
+
+case $PROFILE in
+    "emqx")
+        BROKER="broker"
```
Comment (zmstone) [inline scripts/relup-base-packages.sh:14]:
```suggestion
        S3_DIR="broker"
```

File: scripts/relup-base-packages.sh [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        ARCH='arm64'
+        ;;
+    arm*)
+        ARCH=arm
+        ;;
+esac
+
+SHASUM="sha256sum"
+if [ "$SYSTEM" = "macos" ]; then
+    SHASUM="shasum -a 256"
```
Comment (zmstone) [inline scripts/relup-base-packages.sh:41]: ah, this is why. thank you.

File: .github/workflows/build_packages.yaml (hunk 1 above)
Comment (Rory-Z) [reply .github/workflows/build_packages.yaml:277]: No, in older versions "old_vsn" was only used to pass values, not matrices

File: scripts/relup-base-packages.sh (hunk 2 above)
Comment (Rory-Z) [reply scripts/relup-base-packages.sh]: This is not the path to s3, this is the path to emqx.com, so I've changed it to "DIR"

**Review by zmstone — APPROVED** [review]

**Pull request merged**